### PR TITLE
Starter devel 2

### PR DESCRIPTION
**Pull request by ianabc**
Some changes to the upstream of drupal-vm have broken our current configuration. I'm adding a template config.yml and drupal.make.yml to the starter directory along with a README with some hints for getting things going on OS X.

### CODE REVIEW

File: public/sites/all/modules/contrib/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+backup_migrate
```
Comment (joelpittet) [inline public/sites/all/modules/contrib/.gitignore]: This one is good to have in all environments, probably shouldn't ignore, no?

File: public/sites/all/modules/contrib/.gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+devel
```
Comment (joelpittet) [inline public/sites/all/modules/contrib/.gitignore]: We should have a `public/sites/all/modules/dev/` folder that setup like the `working` folder and we can include development modules in there

**Comment by joelpittet** [comment]
Other than the git ignore this is a great idea. Thanks @ianabc

**Comment by ianabc** [comment]
No problem, I've removed the gitignore, I was just confused because drupalvm was trying to pull in backup_migrate and devel. I've removed them from the drupal.make.yml for the moment, backup_migrate was already included in 2a953abeef10180d0003881460eb037435972509, and I guess devel could go in as another pull request.

**Pull request merged**